.github/pull_request_template.md

```diff
@@ -1,3 +1,8 @@
+<!--
+To mark your pull request as work in progress put the construction emoji 🚧 in your title. (hint: copy it)
```

Comment:
There are also "draft" pull requests (https://github.blog/2019-02-14-introducing-draft-pull-requests/) since February 2019.

Reply:
You cannot turn a PR into a draft PR.

Reply:
For instance, I would love to turn #790 into a draft.

Reply:
@darxriggs see this thread: https://github.community/t5/How-to-use-Git-and-GitHub/Feature-Request-Switch-from-ready-to-draft-in-pull-requests/td-p/19107

Reply:
I'll remove WIP once I can turn a PR into a draft PR :+1:

Reply:
And now we can remove WIP: https://github.blog/changelog/2020-04-08-convert-pull-request-to-draft/